src/datagrid/DataGridValues.tsx

```diff
@@ -387,7 +379,27 @@ export const getPageData = (pageIndex: number, pageSize: number): Promise<DataGr
         }, 2000);
     });
 };
+// Function to get data for page based on pagenumber
+export const getPageDataForSelectedRows = (pageIndex: number, pageSize: number): Promise<DataGridRow[]> => {
+    return new Promise((resolve, reject) => {
+        let rows: DataGridRow[] = [];
 
+        if (pageSize == 5) {
+            if (pageIndex == 2) {
+                rows = alreadySelectedRows.slice(5, 10);
+            }
+            if (pageIndex == 1) {
+                rows = alreadySelectedRows.slice(0, 5);
+            }
+        } else if (pageSize == 10) {
+            rows = paginationRows;
+        }
+        // Purposefully added dealy here to see loading spinner
+        setTimeout(function() {
```

Comment:
Really need this ?  Let the response be quick its fine :)

Reply:
@maheshkulkarni4 I was not able to see loading spinner in pagination story as response is really fast. Hence added this just for story. Keeping this as it is for now.